lib/matplotlib/backend_bases.py

```diff
@@ -531,6 +531,12 @@ def draw_image(self, gc, x, y, im):
 
         *im*
             the :class:`matplotlib._image.Image` instance
+
+        *trans*
+            If `option_scale_image` returns `True`, an affine
+            transformation may also be passed to `draw_image`.  It
+            should be applied to the image, before applying the
+            translation of `x` and `y`.
```

Comment:
What about this?

```
            If the concrete backend is written such that
            `option_scale_image` returns `True`, an affine
            transformation may also be passed to `draw_image`.  The
            backend should apply the transformation to the image
            before applying the translation of `x` and `y`.
```

> Also, "translation" has a very specific meaning that is a subset of "transformation". Was that intentional?

Yes.  The x and y represent the final translation of the image after any transformation given in `trans`.